webpack.config.js

```diff
@@ -65,11 +65,15 @@ const webpackConfig = {
 			{
 				include: /node_modules[\/\\]tinymce/,
 				loader: 'imports?this=>window',
+			},
+			{
+				test: /\.(ml|re)$/,
```

Comment:
Yep! BuckleScript will compile using the `.rei` or `.mli` file, but when referring to the file with `bs-loader` use the implementation